scripts/bash/mg_external.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+set -x
+
+dbglog() {
+    # Allow the input to be piped
+    declare msg=${1:-$(</dev/stdin)}
+
+    echo -e "${msg}" | tee -a "${BASE_COLLECTION_PATH}"/ceph/gather-debug.log
+}
+
+
+# Expect base collection path as an exported variable
+# If it is not defined, use PWD instead
+BASE_COLLECTION_PATH=${1:-"$(pwd)"}
+echo $BASE_COLLECTION_PATH
```

Comment:
it is not must-gather because we only run script.. must-gather is operator
```
dbglog  "ceph commands for external cluster will be collected at ${BASE_COLLECTION_PATH}"
```

Reply:
intention is to add the description for the $BASE_COLLECTION_PATH in echo statement instead of just printing $BASE_COLLECTION_PATH